pkg/bling/BLING_VARS.h

```diff
@@ -274,6 +274,14 @@ C          satellite-equivalent chlorophyll diagnostic (and cost)
      &                     gamma_DON,
      &                     k_Fe_diaz,
      &                     k_NO3,
+#ifdef SIZE_DEP_LIM
+     &                     k_NO3_sm,
+     &                     k_NO3_lg,
+     &                     k_PO4_sm,
+     &                     k_PO4_lg,
+     &                     k_Fe_sm,
+     &                     k_Fe_lg,
+#endif
```

Comment:
I think that we normally have namelist parameters defined independent of the CPP-flag:
```suggestion
     &                     k_NO3_sm,
     &                     k_NO3_lg,
     &                     k_PO4_sm,
     &                     k_PO4_lg,
     &                     k_Fe_sm,
     &                     k_Fe_lg,
```

Reply:
@mjlosch Interesting! I modeled this after pkg/dic, where some namelist parameters are only defined #ifdef DIC_BIOTIC or #ifdef DIC_CALCITE_SAT.

We have a few CPP flags in BLING_VARS.h and bling_readparms.F, e.g. #ifdef USE_SIBLING and #ifdef USE_BLING_V1. Should those be removed too?

Reply:
So this seems to be a BGC-specific method? (Kidding). 

I think it makes a lot of sense to always have the namelist parameters defined, just because the namelist error messages are often very unintuitive, if a parameter is not recognised. Downside: You don't notice that you have accidently forgotten to set a CPP flag, read the namelist and it has no effect. Normally we have some kind of check for this (e.g. check for unset parameters), but this would be very difficult, when you have defaults as in this case. 

In the end, it's up to you, as you and your group are probably the main users of this pkg.